README.rst

```diff
@@ -66,27 +67,70 @@ THE LATEST STABLE VERSION OF NeMo is **0.9.0** (Available via PIP).
 **NeMo Docker Container**
  NVIDIA `NGC NeMo Toolkit container <https://ngc.nvidia.com/catalog/containers/nvidia:nemo>`_ is now available.
 
-* Pull the docker: ``docker pull nvcr.io/nvidia/nemo:v0.9``
+* Pull the docker: ``docker pull nvcr.io/nvidia/nemo:v0.10``
 * Run: ``docker run --runtime=nvidia -it --rm -v <nemo_github_folder>:/NeMo --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/nemo:v0.9``
 
 If you are using the NVIDIA `NGC PyTorch container <https://ngc.nvidia.com/catalog/containers/nvidia:pytorch>`_ follow these instructions
 
 * Pull the docker: ``docker pull nvcr.io/nvidia/pytorch:20.01-py3``
-* Run: ``docker run --runtime=nvidia -it --rm -v <nemo_github_folder>:/NeMo --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/pytorch:20.01-py3``
+* Run:``docker run --runtime=nvidia -it --rm -v <nemo_github_folder>:/NeMo --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/pytorch:20.01-py3``
 * ``apt-get update && apt-get install -y libsndfile1``
-* ``pip install nemo_toolkit`` NeMo core
-* ``pip install nemo_asr`` NeMo ASR (Speech Recognition) collection
-* ``pip install nemo_nlp`` NeMo NLP (Natural Language Processing) collection
-* ``pip install nemo_tts`` NeMo TTS (Speech Synthesis) collection
+* ``pip install nemo_toolkit`` Installs NeMo core only.
+* ``pip install nemo_toolkit[all]`` Installs NeMo core and ALL collections
+* ``pip install nemo_toolkit[asr]`` Installs NeMo core and ASR (Speech Recognition) collection
+* ``pip install nemo_toolkit[nlp]`` Installs NeMo core and NLP (Natural Language Processing) collection
+* ``pip install nemo_toolkit[tts]`` Installs NeMo core and TTS (Speech Synthesis) collection
 
-See `examples/start_here` to get started with the simplest example. The folder `examples` contains several examples to get you started with various tasks in NLP and ASR.
+See `examples/start_here` to get started with the simplest example.
 
 **Tutorials**
 
 * `Speech recognition <https://nvidia.github.io/NeMo/asr/intro.html>`_
 * `Natural language processing <https://nvidia.github.io/NeMo/nlp/intro.html>`_
 * `Speech Synthesis <https://nvidia.github.io/NeMo/tts/intro.html>`_
 
+Fine-tuning using NeMo
```

Comment:
I wouldn't put the TTS models under "Fine-tuning". You can't easily fine-tune Tacotron and you do not need to fine-tune Waveglow

Reply:
Just call this section pre-trained models

Reply:
makes sense - will do

Reply:
done